src/clangd-context.ts

```diff
@@ -18,6 +18,7 @@ export const clangdDocumentSelector = [
   {scheme: 'file', language: 'cuda-cpp'},
   {scheme: 'file', language: 'objective-c'},
   {scheme: 'file', language: 'objective-cpp'},
+  {scheme: 'file', language: 'hlsl'},
```

Comment:
this part is "just code", I think we can make it conditional on a config option.